src/current/v23.2/migration-strategy-live-migration.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Migration Strategy: Live Migration"
+summary: Learn about the 'Live Migration' data migration strategy
+toc: true
+docs_area: migrate
+---
+
+This topic discusses the *live migration* strategy for [migrating your data to CockroachDB]({% link {{ page.version.version }}/migration-overview.md %}). See [Develop a migration plan]({% link {{ page.version.version }}/migration-overview.md %}#develop-a-migration-plan) for detailed information on deciding which migration strategy will work best for your application.
+
+During a live migration you maintain two production databases (the *source* and *target* database) for a period of time, and replicate data between them until a final cutover to the target database. The purpose of a live migration is to minimize downtime during this final cutover period and therefore, maintain service continuity.
+
+A live migration consists of three distinct phases:
```

Comment:
suggest either "the following distinct phases" and using bullets

OR 

"three phases" and using numbered list so each can be referred to by number